docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
   stack:
     image: thethingsnetwork/lorawan-stack
     entrypoint: ttn-lw-stack -c /config/ttn-lw-stack.yml
-    command: start
+    command: start all
```

Comment:
Why? `start` already starts all the basic services right?

Reply:
Reverted

Reply:
Also removed from the getting started guide for consistency